File: tests/Generator/resource/kotlin/complex/Response.kt

import com.fasterxml.jackson.annotation.*

/**
 * Describes the response of the operation
 */
open class Response {
    @JsonProperty("code") var code: Int? = null
    @JsonProperty("contentType") var contentType: String? = null
    @JsonProperty("schema") var schema: PropertyType? = null
}

File: tests/Generator/resource/csharp/complex/Response.cs
using System.Text.Json.Serialization;

namespace TypeAPI.Model;

/// <summary>
/// Describes the response of the operation
/// </summary>
public class Response
{
    [JsonPropertyName("code")]
    public int? Code { get; set; }

    [JsonPropertyName("contentType")]
    public string? ContentType { get; set; }

    [JsonPropertyName("schema")]
    public PropertyType? Schema { get; set; }

}

